build/params_interop.go

```diff
@@ -47,21 +47,24 @@ var UpgradeTurboHeight = abi.ChainEpoch(-15)
 
 var UpgradeHyperdriveHeight = abi.ChainEpoch(-16)
 var UpgradeChocolateHeight = abi.ChainEpoch(-17)
-var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
+var UpgradeOhSnapHeight = abi.ChainEpoch(300)
```

Comment:
This needs to stay disabled -- the FVM can't support v15 (the SnapDeals upgrade yet)